inst/rmarkdown/highlight/a11y.theme

```diff
@@ -0,0 +1,211 @@
+{
+    "text-color": null,
```

Comment:
Good to know ! 
Would it be a good idea to store those `.theme` (and `.css`  equivalent ?) somewhere online ? That would be a lot easier to maintain across packages, using a script to use in dev to update the file. But warning each other of an update works.

(Or this is maybe the kind of resource to live in one package as a dependency for other to use to find the file. )

Reply:
Well there are online in the GitHub repo 😄. But there's some chance we'll move into bslib, since there's general need to match the syntax highlight colour scheme to the rest of the template.

Reply:
It still feel weird to have pandoc highlight files stored in `bslib`. Maybe not all format that will use theme will use bslib. 
Maybe they should live in the "pandoc" package in the future.

Will see.